pygmt/src/basemap.py

```diff
@@ -86,6 +86,7 @@ def basemap(self, projection=None, **kwargs):
     self._activate_figure()
     aliasdict = AliasSystem(
         J=Alias(projection, name="projection"),
+        c=Alias(panel, name="panel", sep=",", size=2),
```

Comment:
I'm think if we should define a function like:
```
def common_alias(value, name):
    match name:
        case "projection":
            return Alias(value, name="projection")
        case "panel":
            return Alias(value, name="panel", sep=",", size=2)
```
then in wrappers, we can use it like:
```
from pygmt.alias import common_alias
aliasdict = AliasSystem(
    J=common_alias(projection, name="projection"),
    c=common_alias(panel, name="panel"),
)
```
so that we don't have to repeat `lias(panel, name="panel", sep=",", size=2),` in all wrappers.